honeybee_energy/material/_base.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, identifier):
         self._locked = False
         self.identifier = identifier
         self._display_name = None
+        
```

Comment:
It looks like you accidentally added an extra blank line here.

Reply:
both refactored and  adressed this additional line
commit:  refatcored materials: I now understand __slots__